README.md

```diff
@@ -30,19 +30,23 @@ make
 ## Usage
 goenvtemplator2 -help
 ```
-Usage of goenvtemplator:
+Usage of goenvtemplator2:
   -debug-templates
-    	Print processed templates to stdout.
+        Print processed templates to stdout.
+  -delim-left {{
```

Comment:
There is `-v int` for verbosity level. Is it correct to say `-delim-left {{`? What I am missing here is the expected data type of the argument. However, `-delim-left charchar` doesn't look good. So whatever, just thinking. @dohnto @FUSAKLA what do you think?

Reply:
Ok seems legit

what about to put in there `-delim-left value` according to the `-env-file value` ?

Reply:
Actually i don't know where exactly those values come from

If you take a look at [this](https://github.com/seznam/goenvtemplator/pull/14/files#diff-e6086192d4941ba49d0ff2187f91badfR84)

I'm not specifying this value and Go or `flag` library somehow completes it from [default value of delimiters of template](https://golang.org/pkg/text/template/#Template.Delims). This is the only way I can imagine how this value gets here.

Reply:
Ok so it looks like I unintentionally used backticks to empathize `{{` and `}}` in flags help which is according to [documentation of flag library](https://golang.org/pkg/flag/#PrintDefaults) parsed out as the example value.

So i removed the backticks and now the example value is `string`.